syntax = "proto3";

package proto;
import "github.com/mwitkow/go-proto-validators/validator.proto";

service CustomerService {
  rpc ListPerson(RequestType) returns (stream Person) {};
  rpc AddPerson(Person) returns (ResponseType) {};
}

message ResponseType {
}

message RequestType {
}

message Person {
  string name = 1;
  int32 age = 2 [(validator.field) = {int_gt: 0, int_lt: 100}];
}
